mcda/utils/utils_for_main.py

```diff
@@ -503,7 +561,6 @@ def check_path_exists(path: str):
     is_exist = os.path.exists(path)
     if not is_exist:
         os.makedirs(path)
-        print("The new output directory is created: {}".format(path))
```

Comment:
If this is removed then probably would be an opportunity to add logs from where this method is called. Also, feels like the need for exception handling?

Reply:
Any comment here?

Reply:
For me, information to the user about where the output files are saved is already given (see comment above). This information contains also implicit information about what output path is used and I wouldn't clutter it more. What do you think?

If the given output path does not exist, then it's created. 

If the user writes junk characters, this issue will be taken care of in the different saving functions (e.g., `save_df`).

Therefore, I am quite satisfied here with the current version.

Reply:
Perfect, thanks.